docs/Roadmap.md

```diff
@@ -0,0 +1,33 @@
+# Roadmap
+
+### Repo Upgrade and Uninstall
+* Support a clear flow to [upgrade](https://github.com/argoproj-labs/argocd-autopilot/issues/45) Argo CD
+* Support a clear flow to [uninstall](https://github.com/argoproj-labs/argocd-autopilot/issues/42) Argo CD
+
+### App Upgrade and Uninstall
+* Support a clear flow to [upgrade](https://github.com/argoproj-labs/argocd-autopilot/issues/44) an app
+* Support a clear flow to uninstall an app
```

Comment:
Shouldn't it be `delete`, to match the command?

Reply:
ah ok changing